src/pages/LandingPages/MGCovid19/MGCovid19.vue

```diff
@@ -50,8 +50,6 @@ export default {
 	},
 	data() {
 		return {
-			isExperimentActive: true,
-			isMonthlyGoodSubscriber: false,
```

Comment:
This data is not used in this template. I also did not modify `MGCovidHero` which does have a MG signup. But I don't think we are going to be using that landing page again.

Reply:
Oh man I sure hope we don't!